app/src/main/java/org/mozilla/focus/widget/AnimatedProgressBar.java

```diff
@@ -325,6 +325,11 @@ private Drawable buildDrawable(@NonNull Drawable original,
         }
     }
 
+    @SuppressWarnings("ResourceType")
+    private boolean isValidInterpolator(@InterpolatorRes int itplId) {
+        return itplId > 0;
```

Comment:
Since the android lint tells us that using ```> 0``` to validate the resource is incorrect, why do we suppress the lint here?

Reply:
My understanding to this is the lint is trying to avoid the situation when a resource should be used by getting the value is misused by using the id directly. However in this case, I believe the dev of this block is trying to use the resource directly to make sure there is such resource. ( >0 )

Reply:
@walkingice Can you share with us what's the rationale behind itplId > 0? Is it a sanity check as I've described?

Reply:
yes it is a checking. But I am not sure whether it is a good practice.

Reply:
Fixed :)